packages/react-native-editor/ios/Podfile.lock

```diff
@@ -518,12 +518,12 @@ SPEC CHECKSUMS:
   React-logger: faee236598b0f7e1a5e3b68577016ac451f1f993
   react-native-blur: ef741a08d020010ba65e411be0ab82d1d325e7ad
   react-native-get-random-values: b6fb85e7169b9822976793e467458c151c3e8b69
-  react-native-keyboard-aware-scroll-view: 0bc6c2dfe9056935a40dc1a70e764b7a1bbf6568
+  react-native-keyboard-aware-scroll-view: 374b637c8684d7de50bd930df1a2d2b8fd49daa5
   react-native-safe-area: c9cf765aa2dd96159476a99633e7d462ce5bb94f
-  react-native-safe-area-context: f0906bf8bc9835ac9a9d3f97e8bde2a997d8da79
+  react-native-safe-area-context: e471852c5ed67eea4b10c5d9d43c1cebae3b231d
   react-native-slider: d4c3367c5db5d2bec3ac0ab9d15ebb73d896034c
-  react-native-video: 0b09cc469df0249509def5dc1484e0419342de9b
-  react-native-webview: 193d233c29eacce1f42ca2637dab7ba79c25a6de
+  react-native-video: 2ad952d1a9ce83e058dcb4a1800af3a99cb7edee
+  react-native-webview: 1f56115845c98f0a59dfbbac685797c014a821be
```

Comment:
I'm wondering why the checksums have changed for these libraries if only we updated the version of `Gutenberg` and `RNTAztecView` 🤔. I've checked the `Podfile.lock` file changes in the previous release ([reference](https://github.com/WordPress/gutenberg/pull/41767/files#diff-ce3a290cf4efc62a6f6097db3b4f49ba4799ca7c66f599bc16d94ed93586028c)) and looks like only `Gutenberg` and `RNTAztecView` changed.

Reply:
> I've updated my PR and it [brings back](https://github.com/WordPress/gutenberg/pull/42009/files#diff-ce3a290cf4efc62a6f6097db3b4f49ba4799ca7c66f599bc16d94ed93586028cL521) these references so I can restore them once I merge it, what do you think?

Yeah, sounds good, let's bring them back on that PR as it will be merged soon.

Reply:
> My understanding is that these types of changes are related to https://github.com/WordPress/gutenberg/pull/32772#discussion_r654569062, and are merely a result of pod install on different development machines with different paths.

Great, thanks for sharing the context. I'm wondering what's the difference between our machines 🤔. I see Ceyhun outlined in [that comment](https://github.com/WordPress/gutenberg/pull/32772#discussion_r654569062) a potential workaround, so it might be interesting to tackle that in the future.

> It might be a good practice to avoid committing this type of change to avoid confusion, however I do not know if one value is "correct."

Yep, it's not clear which values are the "correct" ones. I guess something that could be used as a guide in release PRs is that only `Gutenberg` and `RNTAztecView` Pods should be updated, wdyt?

Reply:
> Yep, it's not clear which values are the "correct" ones. I guess something that could be used as a guide in release PRs is that only `Gutenberg` and `RNTAztecView` Pods should be updated, wdyt?

Yes, that would likely be helpful guidance. Unnecessary `Podfile.lock` changes could still sneak in within non-release PRs, but guidance for releases is likely still beneficial.